content/millennium/r4/security/provenance.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Provenance | R4 API
+---
+
+# Provenance
+
+* TOC
+{:toc}
+
+## Overview
+
+The Provenance resource tracks the source of external data and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter.  Entities are Continuity of Care DocumentReferences (CCDs).
+
+The following fields are returned if valued:
+
+* [Provenance id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Target](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.target){:target="_blank"}
+* [Recorded](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.recorded){:target="_blank"}
+* [Agent type (author or transmitter)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.type){:target="_blank"}
+* [Agent role (source or informant)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.role){:target="_blank"}
+* [Agent who (Practitioner, Device, or Organization)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.who){:target="_blank"}
+* [Agent on behalf of](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.onBehalfOf){:target="_blank"}
+* [Entity role](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.role){:target="_blank"}
+* [Entity what (Continuity of Care DocumentReference id)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.what){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:provenance, :r4) %>
+
+## Search
+
+Search for Provenances that meet supplied query parameters:
+
+    GET /Provenance?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`        | This or `target` | [`token`]     | The logical resource id associated with the resource. Example: `doc-7845`
+ `target`     | This or `_id` | [`reference`] | Target Reference(s) (usually version specific). Example: `DocumentReference/66547`
```

Comment:
Don't we need specify what are all the supported targets?

Reply:
Added [here](https://github.com/cerner/fhir.cerner.com/pull/371/commits/32f7e2bb27361e08c711c456d2481831392d9904#diff-f5afbe5fd0ebfe7f5156f94c1d26876961667b00ce5771a4660569b87d1a661fR45).